src/types.ts

```diff
@@ -29,14 +29,21 @@ export type SignupResponse = {
   evidence: SignupEvidenceSummary
 }
 
-export type RegisterLoginProps = {
-  installationId: string
+type RegisterLoginBaseProps = {
   accountId: string
-  relatedAccountId?: string
   policyId?: string
+  [x: string]: any
+}
+
+export type RegisterLoginProps = RegisterLoginBaseProps & {
+  installationId: string
+  relatedAccountId?: string
   location?: TransactionLocation
   paymentMethodIdentifier?: string
-  [x: string]: any
+}
+
+export type RegisterWebLoginProps = RegisterLoginBaseProps & {
```

Comment:
Web logins don't receive installationId: https://developer.incognia.com/docs/br/v6/apis/login-api#web-login-risk

Is that okay?